elements/pfe-cta/src/pfe-cta.js

```diff
@@ -13,42 +13,94 @@ class PfeCta extends PFElement {
     return "pfe-cta.html";
   }
 
+  get defaultStyle() {
+    return this.hasAttribute("priority") ? false : true;
+  }
+
+  static get observedAttributes() {
+    return [
+      "pfe-priority",
+      "pfe-color",
+      "pfe-alt",
+      "on"
```

Comment:
list should be
```
      "priority",
      "color",
```

Reply:
or did we decide to keep `pfe-` namespacing to avoid conflicts with angular , etc?
@castastrophe @chrisdo1 @markcaron @kylebuch8

Reply:
this issue says keep namespacing https://github.com/patternfly/patternfly-elements/issues/220

Reply:
I'm so torn.  Removing the namspacing is cleaner for developers.  Angular namespaces their stuff with `ng-` though.  I defer to others.  I like it without but can see the value of keeping the namespacing too.

Reply:
Do we need to change the attributes in the light DOM then? 
`<pfe-cta priority="primary"><a href="#">Primary</a></pfe-cta>`
Should be:
`<pfe-cta pfe-priority="primary"><a href="#">Primary</a></pfe-cta>`

Reply:
Since we talked about it as a larger group & it's [currently in the PFE documentation](https://github.com/patternfly/patternfly-elements-documentation/blob/master/content/develop/_index.md#attribute-naming), I think we should follow the prefix recommendation in both the JS file, the light DOM examples, everywhere (find & replace).

Reply:
I want to note that `on=` is the exception since that is being deprecated anyway